rt/cmake/cmake.tdesc

```diff
@@ -0,0 +1,102 @@
+-- -*- lua -*-
+
+-- Skip the CMake test if we can't find a new enough version fo CMake
+local skip_test = true
+local cmake_path = "/bin/false"
+local cmake_ret = os.execute("cmake --version > cmake_version.txt 2>&1")
```

Comment:
Aren't there cleaner ways of checking that? `cmake -E capabilities` or dump a file with
```
cmake_minimum_required (VERSION 3.6.0)
```
and do `cmake -P` with return code check?